config/database.yml

```diff
@@ -11,11 +11,11 @@
 #
 default: &default
   adapter: postgresql
-  pool: <%= ENV.fetch("DB_POOL_SIZE") { ENV.fetch("RAILS_MAX_THREADS") { 5 } * 1.2 } %>
-  username: <%= ENV['DB_USERNAME'] %>
-  password: <%= ENV['DB_PASSWORD'] %>
-  host: <%= ENV['DB_HOST'] %>
-  port: <%= ENV['DB_PORT'] || "5432" %>
+  encoding: utf8
+  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  username: postgres
+  password:
+  host: localhost
```

Comment:
Revert these changes please, we don't want it in version control as part of this PR & it will result in a conflict with some changes I am working on.

Reply:
done